src/components/TravelSection.tsx

```diff
@@ -59,31 +59,79 @@ const TravelSection = () => {
     window.open(link, '_blank', 'noopener,noreferrer');
   };
 
+  const handleKeyDown = (event: React.KeyboardEvent) => {
+    if (event.key === 'ArrowLeft') {
+      event.preventDefault();
+      prevTravel();
+    } else if (event.key === 'ArrowRight') {
+      event.preventDefault();
+      nextTravel();
+    }
+  };
+
+  const formatPrice = (price: number) => {
+    return price.toLocaleString('ko-KR');
+  };
+
   return (
-    <div className={styles.travelSection}>
-      <button className={`${styles.navButton} ${styles.navButtonPrev}`} onClick={prevTravel}>
-        <img src={chevronLeft} className={styles.navButtonIcon} />
+    <div
+      className={styles.travelSection}
+      role="region"
+      aria-label="지금 떠나기 좋은 여행 상품 캐로셀"
+      onKeyDown={handleKeyDown}
+      tabIndex={0}
+    >
+      <button
+        className={`${styles.navButton} ${styles.navButtonPrev}`}
+        onClick={prevTravel}
+        type="button"
+      >
+        <img src={chevronLeft} className={styles.navButtonIcon} alt="이전 여행 상품" />
       </button>
-      <div className={styles.carousel}>
+
+      <div className={styles.carousel} role="group" aria-live="polite">
+        <div className={styles.srOnly}>
+          {travelOptions.length}개의 여행상품중 {currentIndex + 1}번째 상품
+        </div>
         {travelOptions.map((option, index) => (
           <div
             key={index}
             className={`${styles.card} ${index === currentIndex ? styles.cardActive : ''}`}
             onClick={() => handleCardClick(option.link)}
+            onKeyDown={(e) => {
+              if (e.key === 'Enter' || e.key === ' ') {
+                e.preventDefault();
+                handleCardClick(option.link);
+              }
+            }}
+            role="button"
+            tabIndex={index === currentIndex ? 0 : -1}
+            aria-label={`${option.departure} 출발 ${option.destination} 도착 ${
+              option.type
+            } 가격 ${formatPrice(option.price)}원 선택하면 예약 페이지로 이동합니다`}
           >
-            <img src={option.image} className={styles.cardImage} />
+            <img
+              src={option.image}
+              className={styles.cardImage}
+              alt={`${option.departure}에서 ${option.destination}으로 가는 여행 상품 이미지`}
+            />
```

Comment:
```suggestion
            <img
              src={option.image}
              className={styles.cardImage}
              alt=""
            />
```
`img` 태그는 스크린 리더가 기본적으로 이미지라는 것을 인식하고 안내해줘서 `alt` 속성에 `~~이미지`라는 표현을 사용하는 것은 중복된 표현이고, `~에서 ~으로 가는`이라는 표현도 위 `aria-label`에서 안내해주고 있기 때문에 중복된 표현인 것 같아서 제거해줘도 좋을 것 같네요🙃

Reply:
https://github.com/woowacourse/a11y-airline/pull/155/commits/47f23b5099c75915bd25469f975f939d4bbaff5f

그렇네요!! 불필요한 alt 문을 제거하였습니다!